server/command.go

```diff
@@ -341,6 +357,23 @@ func (p *Plugin) webhookCommand(parameters []string, info *gitlab.UserInfo) stri
 		}
 		return fmt.Sprintf("Invalid command: %s", subCommand)
 
+	case commandBuild:
+		if len(parameters) != 3 {
+			return unknownActionMessage
+		}
+		project := parameters[1]
+		ref := parameters[2]
+
+		pipeline, err := p.GitlabClient.TriggerNewBuildPipeline(info, project, ref)
+		if err != nil {
+			if strings.Contains(err.Error(), projectNotFoundError) {
+				return projectNotFoundMessage + project
+			}
+			return err.Error()
+		}
+
+		return pipeline.String()
```

Comment:
Does this code duplicate L234ff ?

Reply:
yes it does, I tried to run it outside of webhook but that does not work for me, So i'll remove the L234 code and keep this one.